tests/modules/asset_groups/resources/test_commit_asset_group.py

```diff
@@ -12,70 +11,59 @@
 @pytest.mark.skipif(
     module_unavailable('asset_groups'), reason='AssetGroups module disabled'
 )
-def test_commit_asset_group(flask_app_client, researcher_1, regular_user, test_root, db):
+def test_commit_asset_group(
+    flask_app_client, researcher_1, regular_user, test_root, request, db
+):
     # pylint: disable=invalid-name
-    from tests.modules.asset_groups.resources.utils import AssetGroupCreationData
     from app.modules.sightings.models import Sighting, SightingStage
 
-    transaction_id, test_filename = tus_utils.prep_tus_dir(test_root)
-    asset_group_uuid = None
-    try:
-        data = AssetGroupCreationData(transaction_id)
-        data.add_filename(0, test_filename)
-        response = asset_group_utils.create_asset_group(
-            flask_app_client, regular_user, data.get()
-        )
-        asset_group_uuid = response.json['guid']
-        asset_group_sighting_guid = response.json['asset_group_sightings'][0]['guid']
-        asset_uuid = response.json['assets'][0]['guid']
-        asset_group_utils.patch_in_dummy_annotation(
-            flask_app_client, db, researcher_1, asset_group_sighting_guid, asset_uuid
-        )
-
-        # Should not be able to commit as contributor
-        asset_group_utils.commit_asset_group_sighting(
-            flask_app_client, regular_user, asset_group_sighting_guid, 403
-        )
-        # researcher should though
-        response = asset_group_utils.commit_asset_group_sighting(
-            flask_app_client, researcher_1, asset_group_sighting_guid
-        )
-
-        sighting_uuid = response.json['guid']
-        sighting = Sighting.query.get(sighting_uuid)
-        assert sighting.stage == SightingStage.un_reviewed
-        assert len(sighting.get_encounters()) == 1
-        assert len(sighting.get_assets()) == 1
-        assert sighting.get_owner() == regular_user
-        group_sighting = asset_group_utils.read_asset_group_sighting(
-            flask_app_client, researcher_1, asset_group_sighting_guid
-        )
-        assert set(group_sighting.json.keys()) >= set(
-            {
-                'completion',
-                'asset_group_guid',
-                'assets',
-                'stage',
-                'creator',
-                'jobs',
-                'config',
-                'guid',
-                'sighting_guid',
-            }
-        )
-        assert group_sighting.json['completion'] == 76
-        assert group_sighting.json['asset_group_guid'] == asset_group_uuid
-        assert group_sighting.json['creator']['guid'] == str(regular_user.guid)
-        assert group_sighting.json['sighting_guid'] == sighting_uuid
-
-    finally:
-        # Restore original state
-        if asset_group_uuid:
-            asset_group_utils.delete_asset_group(
-                flask_app_client, regular_user, asset_group_uuid
-            )
-
-        tus_utils.cleanup_tus_dir(transaction_id)
+    (
```

Comment:
like many changes, this is the use of the create_simple_asset_group which does the cleanup, hence no need for the try, finally